packages/core/manifest/backend.yml

```diff
@@ -0,0 +1,64 @@
+name: Blog about cats
```

Comment:
@SebConejo this file should be in the `manifest` folder to be taken in consideration. If it is unused you can delete it. 

The one in the manifest folder in in gitignore to prevent commiting it.